e2e/commands_test.go

```diff
@@ -367,3 +373,10 @@ func checkContains(t *testing.T, combined string, expectedLines []string) {
 		assert.Assert(t, exp.MatchString(combined), expected, combined)
 	}
 }
+
+func runCmdOnTmpDir(t *testing.T, tmpDir *fs.Dir, command ...string) *icmd.Result {
+	return icmd.RunCmd(icmd.Cmd{
```

Comment:
You can also create one command, and re-use it when running a new command, like we did with https://github.com/docker/app/blob/master/e2e/commands_test.go#L283